routes/curations.js

```diff
@@ -52,10 +52,51 @@ router.get(
 router.get(
   '/:type?/:provider?/:namespace?/:name?',
   asyncMiddleware(async (request, response) => {
-    const coordinates = utils.toEntityCoordinatesFromRequest(request)
-    return curationService.list(coordinates).then(result => response.status(200).send(result))
-  })
-)
+      const coordinates = utils.toEntityCoordinatesFromRequest(request)
+      curationService.list(coordinates).then(function(result){
+
+        // access the request url and method
+        const { method, url } = request;
+        console.log('method', method);
+        console.log('url', url);
+
+        //trim and split the request url for processing
+        let trimmed = url.trim();
+        let myStringArray = trimmed.split("/");
+        let text_file_name = "./test/fixtures/dummy_data_curations_"
+
+        //iterate through the 'url array' and create a filename string from it
+        const arrayLength = myStringArray.length;
+        for (var i = 0; i < arrayLength; i++) {
+              if((myStringArray[i] === "-") || (myStringArray[i] === "")) {
+                //strip out the spaces and dashes
+                console.log('space or dash stripped out');
```

Comment:
Are we missing an instruction here, other than console log?
If we just need to log stuffs, maybe we can avoid this particular if condition, unless is of vital importance.

Reply:
Agreed. I've refactored this out into it's own method and modified it.